### DISCUSSION TITLE
firefox: no network access on Suse Tumbleweed (resolv.conf)

### DISCUSSION THREAD

**Discussion by GrislyTiger** [netblue30/firejail#4987] [Q&A] [answered]
Hi all. I am no coder but have been running firefox inside firejail sandboxes. But since firejail was upgraded from 0.9.66 to 0.9.68 I get the "server not found" message when trying a web site from firefox inside firejail (it starts fine). That was actually on Suse Leap 15.3 and I am still using it there by rolling back to the previous (0.9.66) release but I am looking for help to continue using firejail in Suse; I got no suggestions in the Suse forums so I'm guessing this is not a simple user error. 

I am also running Suse Tumbleweed which has Firejail 0.9.68 (same problem). It is a few weeks since the thread about the release of 0.9.68, so am starting a new thread. Any suggestions? Perhaps I need a new network stack? I have uncommented the line in firejail.conf (all the lines were commented out) about allowing networks to no effect.

**Comment by rusty-snake** [comment, marked as answer]
Duplicate of #4954


Add to `/etc/firejail/whitelist-run-common.inc`:
```
whitelist /run/netconfig/resolv.conf
```
